src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int randomIndex = random.nextInt(Colors.values().length);
+        return Colors.values()[randomIndex].name();
```

Comment:
`Colors.values()` call result can be saved to a variable to avoid multiple calls